ShopGo/Create Account/CreateAccountViewController.swift

```diff
@@ -24,10 +29,10 @@ class CreateAccountViewController: UIViewController {
     }
     
     func setup() {
```

Comment:
make a separate extension file for all such functions that don't need to be on the view controller to keep it as clean as possible. At the moment wait for tonight as I have created a separate class for the UI button which everyone can use instead of making a new button everytime.